modules/controller_pool/controller-primary.tpl

```diff
@@ -238,6 +238,7 @@ stringData:
     {
       "apiKey": "${equinix_api_key}",
       "projectID": "${equinix_project_id}",
+      %{ if metro > ""}"metro": "${equinix_metro}"%{ else }"facility": "${equinix_facility}"%{ endif },
```

Comment:
I was about to open a similar PR. Just pushed this `facility` addition into the branch along with enabling CPEM from examples/simple.